.doks/doks-cluster/databases.tf

```diff
@@ -0,0 +1,67 @@
+######################
```

Comment:
It's better, less repeated stuff. 
Still, I would put this in a different directory than "doks-cluster" (it isn't a doks cluster).

After that you can just re-use it inside circleci wth `terraform apply database/ -var 'name=codacy-postgres-$ENVIRONMENT'`

With this I can make changes to the database without having to do in all of them at the same time

Reply:
Everything digital ocean is now on the same workspace. Which means that first I will move the db related resources to a separate workspace, since moving the files to a separate folder will cause some resource definitions (k8s cluster for example) to be absent. Meaning, terraform would just want to delete our k8s cluster.

The main problem now is that, because we decided to go with a non-declarative approach (with nested for loops to reduce the amount of repetition), we can not apply individual configurations to the db clusters. Say that you want to have a better instance type for the release environment. Or that you want to bump sandbox to postgres11 for testing. We can not do that.
Unless what you are meaning is to be able to provision db clusters a-la-carte, to create a brand new environment for example. That's valuable but is completely out of the scope of the original ticket which was just to bump postgres on digital ocean to 10. :'(